spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -69,7 +69,7 @@ deployment to fail
 
 [subs=attributes]
 ```
-cf push dataflow-server -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
+cf push dataflow-server -b java_buildpack_offline -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
```

Comment:
That was one of my concern, that names are not consistent.
Tested on our GCP env.

Not sure there is one "that comes from default" actually... Maybe this is not worth the trouble after all. WDYT @markpollack ?